managed-databases/document-database/how-to/create-a-database-instance.mdx

```diff
@@ -0,0 +1,56 @@
+---
+meta:
+  title: How to create a Document Database Instance
+  description: This page explains how to create a Document Database Instance
+content:
+  h1: How to create a Document Database Instance
+  paragraph: This page explains how to create a Document Database Instance
+tags: postgresql ferretdb mongodb database-instance managed-database database
+dates:
+  validation: 2023-08-01
+  posted: 2023-08-01
+categories:
+  - managed-databases
+---
+
+Managed Document Databases provide fully-managed document Database Instances, with FerretDB as a database engine.
+
+Document databases enable users to store and retrieve data in a document format, such as `json`. Compared to traditional relational databases where data is stored in a table-like format, document-type storage supports storing multiple nested keys and values in each document key.
+
+<Macro id="iam-requirements" />
+
+<Message type="requirement">
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
+</Message>
+
+1. Click **Document Databases** under **Managed Databases** on the side menu, if you do not have a Database Instance already created, the creation page displays.
+2. Click **Create a Database Instance**. The creation wizard displays.
+3. Complete the following steps in the wizard:
+    - Select **FerretDB** as a database engine.
+    - Choose a region. This is the geographical location where your database will be deployed.
+    - Select a database configuration. You can choose between:
+        - **High Availability**: creates a secondary node with an up-to-date replica of the database. If the primary Instance fails, the secondary takes over requests.
+        - **Standalone**: creates a standalone database provisioned on a single node.
+    - Select a node type.
+    - Define the size of your Block Storage volume. You can increase your storage space without changing your node type, with no downtime. You can increase your volume to up to 50 GB.
```

Comment:
"volume to up to 50 GB" -> "volume to up to 10 TB" (like managed db for postgresql)